apps/backend/src/lambdas/utils/eventTypes.ts

```diff
@@ -1,63 +1,66 @@
-// From: https://gist.github.com/jeshan/52cb021fd20d871c56ad5ce6d2654d7b
-
+// From https://gist.github.com/jeshan/52cb021fd20d871c56ad5ce6d2654d7b
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const getLambdaEventSource = (event?: any) => {
-  if (event.Records && event.Records[0].cf) return 'isCloudfront';
+  switch (event) {
+    case event?.Records[0]?.cf:
+      return 'isCloudfront';
 
-  if (event.configRuleId && event.configRuleName && event.configRuleArn)
-    return 'isAwsConfig';
+    case event.configRuleId && event.configRuleName && event.configRuleArn:
+      return 'isAwsConfig';
 
-  if (event.Records && event.Records[0].eventSource === 'aws:codecommit')
-    return 'isCodeCommit';
+    case event.Records && event.Records[0].eventSource === 'awscodecommit':
```

Comment:
I see we removed the colon from all of these, was this broken before?

Reply:
I'm pretty confused at the scope of event types we're triggering, there are quite a few services here that we don't have provisioned - do we have any history on whether lambda jobs actually trigger things like sqs?

Reply:
This was not broken, but sonarqube complained about the complexity. We're not actually using this anywhere so it could be removed.

Reply:
Logged [CCFPCM-617](https://bcdevex.atlassian.net/browse/CCFPCM-617)